### PR TITLE
Project - 5

### PR DESCRIPTION
**Pull request by natalieschnelzer**
Changes in javascript, css and html stay the same

### CODE REVIEW

File: code/clue.js [hunk 1]
```diff
@@ -1,64 +1,179 @@
...
+  'study',
+  'hall',
+  'lounge',
+  'dining room'
 ]
 
-const weapons = []
+let windowKiller = null 
+let windowWeapon = null
+let windowRoom = null
```
Comment (JennieDalgren) [inline code/clue.js:123]: how come you removed the mystery object?
This is fine, I'm just curious.

File: code/clue.js [hunk 2]
```diff
@@ -1,64 +1,179 @@
...
 
-const rooms = []
 
-// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FOR THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-const randomSelector = array => {
-  return array[Math.floor(Math.random() * array.length)]
+// SELECTING RANDOM 
+function randomOptionSelector (optionsArray) {
+  const randomIndex = Math.floor(Math.random() * optionsArray.length)
+  return optionsArray[randomIndex]
```
Comment (JennieDalgren) [inline code/clue.js:129]: how come you moved away from es6 arrow function syntax?

File: code/clue.js [hunk 3]
```diff
@@ -1,64 +1,179 @@
...
 }
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS THE THE WEAPONS IF YOU LIKE.
+const mrsWhite = {
+  title: "Mrs",
+  lastName: "White",
+  color: "white",
+  description: "Frazzled",
+  age: 54,
+  image: "assets/white.png",
```
Comment (JennieDalgren) [inline code/clue.js:19]: would be cool to see if you can get the images to show on the cards as well? 😎

**Review by JennieDalgren — APPROVED** [review]

File: code/clue.js (hunk 1 above)
Comment (natalieschnelzer) [reply code/clue.js:123]: Tiago was helping me understand the flow so we changed the names and broke it down in smaller pieces.

File: code/clue.js (hunk 2 above)
Comment (natalieschnelzer) [reply code/clue.js:129]: Also, a way to understand and break it down a little bit further. Will be going through the code again and double-check these things and try to understand them for es6

File: code/clue.js (hunk 3 above)
Comment (natalieschnelzer) [reply code/clue.js:19]: Yes!! It's on my to-do list!

**Pull request closed without merging**